### ISSUE TITLE
The clustering results are improved when adversarial module is removed

### ISSUE THREAD

**Issue by dawnranger** [Ruiqi-Hu/ARGA#4]
In the paper, the reported results of clustering on cora dataset are : 
```
Cora     Acc     NMI     F1    Precision    ARI
ARGE    0.640   0.449   0.619    0.646     0.352
ARVGE   0.638   0.450   0.627    0.624     0.374
```
In the experiment section, ARGA is not compared with gae. When I remove the adversarial module and use simple gae(not vae), the results on cora dataset look like this:
```
Epoch: 0030 ACC=0.6403 f1_macro=0.6222 f1_micro=0.6403 NMI=0.4732 ADJ_RAND_SCORE=0.4154
Epoch: 0032 ACC=0.6448 f1_macro=0.6264 f1_micro=0.6448 NMI=0.4796 ADJ_RAND_SCORE=0.4211
Epoch: 0034 ACC=0.6470 f1_macro=0.6276 f1_micro=0.6470 NMI=0.4782 ADJ_RAND_SCORE=0.4256
Epoch: 0036 ACC=0.6536 f1_macro=0.6362 f1_micro=0.6536 NMI=0.4809 ADJ_RAND_SCORE=0.4264
Epoch: 0038 ACC=0.6558 f1_macro=0.6397 f1_micro=0.6558 NMI=0.4786 ADJ_RAND_SCORE=0.4245
Epoch: 0040 ACC=0.6603 f1_macro=0.6435 f1_micro=0.6603 NMI=0.4840 ADJ_RAND_SCORE=0.4305
Epoch: 0042 ACC=0.6448 f1_macro=0.6356 f1_micro=0.6448 NMI=0.4771 ADJ_RAND_SCORE=0.4081
Epoch: 0044 ACC=0.6514 f1_macro=0.6430 f1_micro=0.6514 NMI=0.4745 ADJ_RAND_SCORE=0.4051
Epoch: 0046 ACC=0.6518 f1_macro=0.6433 f1_micro=0.6518 NMI=0.4786 ADJ_RAND_SCORE=0.4088
Epoch: 0048 ACC=0.6629 f1_macro=0.6464 f1_micro=0.6629 NMI=0.4655 ADJ_RAND_SCORE=0.4236
Epoch: 0050 ACC=0.6510 f1_macro=0.6404 f1_micro=0.6510 NMI=0.4722 ADJ_RAND_SCORE=0.4075
```

It seems that the adversarial module doesn't help much to improve the performance?

**Comment by Ruiqi-Hu** [comment]
1. The results reported in our paper are the average scores after ten times experiments without any seed.
We set a universal seed for all experiments and all datasets in our source code in Github, just for the conveniently repeating our results.
BTW, we did compare our algorithms with GAE in our paper.

2. I repeated the experiments as you described, and got the results all with the same setting except the adversarial module, of course:

**WITH** the adversarial module, Cora, ae, clustering:
```
ACC=0.703102, f1_macro=0.687405, precision_macro=0.690322, recall_macro=0.712566, f1_micro=0.703102, precision_micro=0.703102, recall_micro=0.703102, NMI=0.483894, ADJ_RAND_SCORE=0.459092 
```
**WITHOUT** the adversarial module, Cora, ae, clustering:
```
ACC=0.631093, f1_macro=0.620762, precision_macro=0.629933, recall_macro=0.654234, f1_micro=0.631093, precision_micro=0.631093, recall_micro=0.631093, NMI=0.439196, ADJ_RAND_SCORE=0.381250 
```

3. The results from many clustering algorithms, including k-means, varies because of some random factors, like the seed. So the reported results are supposed to be the average ones after multi times running.
